tests/locust_stress_testing.py

```diff
@@ -140,7 +140,7 @@ class DummyManager(DummyClientLocust):
 
     host = HOST
     port = PORT
-    wait_time = lambda x: 3
+    wait_time = lambda x: 20
```

Comment:
Did this work? Doesn't this mean less stress? But on the other hand, will the problem that this tries to solve reappear if we add clients again?

Reply:
It increases the wait time between new tasks, which I think does increase the space for clients to finish a full task before the broker is bombarded.

But yeah, it does mean less stress. I do not think there is a better way for us to address this. =( There is a tension between adding clients, increasing client activity, and making sure clients have space to send states upon request.

Reply:
We can also make this an actual function, where a random wait time in a range is chosen. That might be a good middle ground.